### ISSUE TITLE
Exceptions when switching to any theme

### ISSUE THREAD

**Issue by gene-pavlovsky** [fdorg/flashdevelop#1108]
Just installed the latest FD 5.1 release, and I have the following problem:
When I select a theme (any theme, including the stock themes supplied with FD5.1), whether a UI theme, syntax theme, or a full theme, I get an exception pop up:

```
Value cannot be null.
Parameter name: value

   at System.Windows.Forms.ImageList.ImageCollection.Add(Image value)
   at System.Windows.Forms.ImageList.ImageCollection.AddRange(Image[] images)
   at ProjectManager.Controls.Icons.FDImageList.OnRefresh()
   at PluginCore.Managers.ImageListManager.RefreshAll()
   at FlashDevelop.MainForm.AdjustAllImages()
   at FlashDevelop.Managers.ThemeManager.WalkControls(Object obj)
```

If I do select a theme one more time, there's a different exception:

```
Index was outside the bounds of the array.

   at ProjectManager.Controls.Icons.FDImageList.OnRefresh()
   at PluginCore.Managers.ImageListManager.RefreshAll()
   at FlashDevelop.MainForm.AdjustAllImages()
   at FlashDevelop.Managers.ThemeManager.WalkControls(Object obj)
```

Other than the exceptions, looks like the theme is applied and FD continues to work normally.
I've had these plug-ins:
```
AutohideMenu.dll
AutoVersion.dll
CompactMax.dll
InterfaceSuggest.dll
OpenTheDoc.dll
```
I've removed all of them and restarted FD but still have the exceptions thrown.

I then closed the project I've had open (an AIR Mobile app), and restarted, after that applying theme throws no exception. Reopened the project and applied theme - exceptions again! The icons on the project panel are all messed up, here's a screenshot:

![untitled-1](https://cloud.githubusercontent.com/assets/2841193/13555591/6b1a38c4-e3d5-11e5-8e87-c9c0a0730839.png)

FD 5.1.0.3, .NET 3.5.30729.5420

**Comment by gene-pavlovsky** [comment]
Additionally, opening a file from the project panel throws another exception.
InvalidArgument=Value of '68' is not valid for 'index'.
Parameter name: index

   at System.Windows.Forms.ImageList.ImageCollection.get_Item(Int32 index)
   at ProjectManager.PluginMain.SetDocumentIcon(ITabbedDocument doc)
   at ProjectManager.PluginMain.HandleEvent(Object sender, NotifyEvent e, HandlingPriority priority)
   at PluginCore.Managers.EventManager.DispatchEvent(Object sender, NotifyEvent e)

**Comment by Meychi** [comment]
Thanks for reporting, fixed in 003a0cf98302a5db2225fe1a282a149caaf05b8b

**Comment by gene-pavlovsky** [comment]
Thanks, confirmed.
